devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -52,6 +52,7 @@ metadata {
         fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,0300,1000", outClusters: "0019", manufacturer: "Philips", model: "LST002", deviceJoinName: "Philips Hue Lightstrip"
         fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Smart Bulb Color"
         fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "FL 130 C", deviceJoinName: "Innr Flex Light Color"
+        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF", outClusters: "0019",manufacturer: "Megaman", model: "ZLL-ExtendedColor", deviceJoinName: "INGENIUM ZB RGBW Light"
```

Comment:
missing a space before `manufacturer`. if this was auto-generated it needs to be addressed with the dev workspace team

Reply:
A space before manufacturer was added

Reply:
@greens I could not re-submit the PR and always shown "There was an error submitting your request" on smartThings IDE.
Please advice, thanks!

Reply:
@greens this issue resolved, thanks!